### PR TITLE
Ms 35488: : Updated Editing the Default State Instance and Customizing Provisioning States sections

### PR DESCRIPTION
**Pull request by meghasalunke1**
Steps are updated as suggested in [this](https://github.ibm.com/katamari/dev-issue-tracking/issues/35488#issuecomment-52090019) issue.

### CODE REVIEW

**Comment by meghasalunke1** [comment]
@miq-bot add_label bug

**Comment by meghasalunke1** [comment]
@miq-bot add-label oparin/yes?

**Comment by meghasalunke1** [comment]
@miq-bot assign @Fryguy

File: provisioning_virtual_machines_and_hosts/_topics/fulfilling_requests.md [hunk 1]
```diff
@@ -175,27 +175,35 @@ Grayed out items reflect values that are set in the class schema. These values c
...
 
-1.  Navigate to menu:Automate\[Explorer\].
+1.  Navigate to menu:Automation \> Embedded Automate \> Explorer.
 
-2.  From the accordion menu, click menu:DOMAIN\[Cloud \> VM \> Provisioning \> StateMachines \> VMProvision\_VM\].
+2.  Click **Datastore** accordion. Select **Configuration** > Add a New Domain. You must fill the Name and Description values and click **Add** to create a custom 
+    domain.
 
-    **Note:**
+3.  From the accordion menu, navigate to ManageIQ \> Cloud \> VM \>Provisioning \> StateMachines \> VMProvision_VM.
```
Comment (Fryguy) [inline provisioning_virtual_machines_and_hosts/_topics/fulfilling_requests.md:187]: We should avoid `ManageIQ` in the docs, as it affects productization

```suggestion
3.  From the accordion menu, navigate to Cloud \> VM \>Provisioning \> StateMachines \> VMProvision_VM.
```

**Comment by meghasalunke1** [comment]
@Fryguy Steps are updated as suggested in [this](https://github.ibm.com/katamari/dev-issue-tracking/issues/35488#issuecomment-52090019) issue.

File: provisioning_virtual_machines_and_hosts/_topics/fulfilling_requests.md (hunk 1 above)
Comment (meghasalunke1) [reply provisioning_virtual_machines_and_hosts/_topics/fulfilling_requests.md]: Done

**Comment by Fryguy** [comment]
Backported to `oparin` in commit c6d5b0fdd8198a564d5a6ec860e9341ed172eb53.

```text
commit c6d5b0fdd8198a564d5a6ec860e9341ed172eb53
Author: Jason Frey <fryguy9@gmail.com>
Date:   Mon Feb 13 12:21:36 2023 -0500

    Merge pull request #1719 from meghasalunke1/MS_35488
    
    Ms 35488: : Updated Editing the Default State Instance and Customizing Provisioning States sections
    
    (cherry picked from commit e5a871cfca34a4ff4869d44e58078282f72a9520)
```

**Pull request merged**